src/Jackett.Common/Definitions/torrentseeds.yml

```diff
@@ -0,0 +1,196 @@
+---
```

Comment:
actually, for new unit3d we want to drop login support.
we only continue the login method for existing sites, but use the api going forward.